patches/0004-Add-OpenSSL-crypto-backend.patch

```diff
@@ -714,29 +714,29 @@ index c83a7272c9f01f..a0548a7f9179c5 100644
  package x509
  
 diff --git a/src/go.mod b/src/go.mod
-index b38dee7e6ddc28..097f35ca01aa5b 100644
+index 789f5aaa1d3e26..2f397cf20abb0a 100644
 --- a/src/go.mod
 +++ b/src/go.mod
 @@ -3,6 +3,7 @@ module std
  go 1.23
  
  require (
-+	github.com/golang-fips/openssl/v2 v2.0.3
++	github.com/golang-fips/openssl/v2 v2.0.4-0.20240905155948-17d05d3f692c
  	golang.org/x/crypto v0.23.1-0.20240603234054-0b431c7de36a
  	golang.org/x/net v0.25.1-0.20240603202750-6249541f2a6c
  )
 diff --git a/src/go.sum b/src/go.sum
-index c5eab7d4b2f93b..fac8a0626146bc 100644
+index a75ea98c7312df..7729cca493647c 100644
 --- a/src/go.sum
 +++ b/src/go.sum
 @@ -1,3 +1,5 @@
-+github.com/golang-fips/openssl/v2 v2.0.3 h1:9+J2R0BQio6Jz8+dPZf/0ylISByl0gZWjTEKm+J+y7Y=
-+github.com/golang-fips/openssl/v2 v2.0.3/go.mod h1:7tuBqX2Zov8Yq5mJ2yzlKhpnxOnWyEzi38AzeWRuQdg=
++github.com/golang-fips/openssl/v2 v2.0.4-0.20240905155948-17d05d3f692c h1:q7tCgQctS2aXmDVWjTV0951iRioF7Svx/hSQxymkZYo=
```

Comment:
We seemed to go from a simple tag of 2.0.3 - just making sure that the new 2.0.4+hash tag is what we really wanted here.

Reply:
Yeah, this is a move from the `v2` branch, which has tags, to the `ms-go1.23-support` branch, which doesn't, and in that case [Go makes up a pseudo-version](https://go.dev/ref/mod#pseudo-versions).